Source/DafnyPipeline.Test/PluginsTest.cs

```diff
@@ -45,71 +47,49 @@ public class PluginsTest {
     return assemblyPath;
   }
 
-  class CollectionErrorReporter : BatchErrorReporter {
```

Comment:
Makes sense to delete it. I wrote It originally because the BatchErrorReporter's error messages were private. Now that they are accessible, this class is no longer needed.